googlemaps/distance_matrix.py

```diff
@@ -75,6 +75,10 @@ def distance_matrix(client, origins, destinations,
         requests. Valid values are "less_walking" or "fewer_transfers".
     :type transit_routing_preference: string
 
+    :param traffic_model: Specifies the predictive travel time model to use.
```

Comment:
As an aside, I'm OK with it not being implemented in code, since it strikes me as the kind of thing that may change on the server at some point, and could theoretically change from client to client.